backend/internal/application/store.go

```diff
@@ -109,7 +110,7 @@ func (st *applicationStore) CreateApplication(app model.ApplicationProcessedDTO)
 
 // GetTotalApplicationCount retrieves the total count of applications from the database.
 func (st *applicationStore) GetTotalApplicationCount() (int, error) {
-	dbClient, err := provider.GetDBProvider().GetDBClient("identity")
+	dbClient, err := provider.GetDBProvider().GetDBClient(database.IdentityDBClientName)
```

Comment:
Shall we have separate functions ? rather than passing the database name as argument, because it's not gonna be much dynamic

```
s.dbProvider.GetUserDBClent()
s.dbProvider.GetRuntimeDBClent()
s.dbProvider.GetConfigDBClent()
```